tests/images.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { test } from "./fixtures/lxd-test";
+import { deleteInstance, randomInstanceName } from "./helpers/instances";
+
+test("search for custom image and create an instance from it", async ({
```

Comment:
this test would fail locally I think, maybe we should skip it for dev environment? Or we could setup a shell script to run before the e2e tests kicks off?

Reply:
Right, this hinges on the availability of the custom image.
I was planning to extend this test to create the custom image from the UI in the next cycle (as we have the item to add support for that on our roadmap). Maybe we can ignore this incompatibility for local test runs for now and manually create the custom image on our dev machines?

Reply:
Cool sounds good :+1: